src/wrappers/themis/jsthemis/secure_message.hpp

```diff
@@ -38,6 +38,8 @@ namespace jsthemis{
 
     static Nan::Persistent<v8::Function> constructor;
 
+    static bool ValidateKeys(const std::vector<uint8_t>& private_key, const std::vector<uint8_t>& public_key);
```

Comment:
imho would be better to move these function to `keys.hpp` or something like that and separate from secure message because it is useful for secure session too. and if I will try to find this function in github without search then I try to find some `key*.h` file or `utils.h`/`keygen.h` or something unrelated with specific component.
Plus unobvious why wrapper for this function in jsthemis related with SecureMessage class and not with SecureSession... Better to move it to general jsthemis/themis namespace imho

Reply:
Makes sense. You also reminded me that Secure Session uses private and public keys as well so we should probably add checks there too.

Reply:
I just realized that my view of Secure Session may be incorrect. I thought it supports only ECDSA keys, but it may turn out that RSA keys are fine too. I'd like to do changes in Secure Session separately after we know what to expect from the keys.

However, I'll move key generation and validation routines to separate modules in this PR, since we're already touching this API.